Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/common/DccTicketingJwtException.kt

```diff
@@ -0,0 +1,15 @@
+package de.rki.coronawarnapp.dccticketing.core.common
+
+class DccTicketingJwtException(
+    val errorCode: ErrorCode,
+    override val cause: Throwable? = null
+) : Exception(errorCode.message, cause) {
+
+    enum class ErrorCode(val message: String) {
+        JWT_VER_NO_JWKS("JWT_VER_NO_JWKS"),
```

Comment:
Not sure why we need the same code in the message. I would prefer a short description of the error (as it was before)

Reply:
I'm going to clean up errors in another task https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10648